### PR TITLE
Fix e2fsck version issue

### PR DESCRIPTION
**Pull request by wanxiangyuer**

### CODE REVIEW

File: zthin-parts/zthin/bin/unpackdiskimage [hunk 1]
```diff
@@ -305,7 +305,10 @@ function resizePartition {
     #Resize the ext filesystem.
     out=`e2fsck -yf ${lastPartition} 2>&1`
     rc=$?
-    if (( rc != 0 && rc != 1 && rc != 2 )); then
+
+    #Considering that the e2fsck version of rhel77 is too low to support 
+    #the rhel8 guest VM, so the return value of 8 is ignored.
+    if (( rc != 0 && rc != 1 && rc != 2 && rc != 8 )); then
```
Comment (jichenjc) [inline zthin-parts/zthin/bin/unpackdiskimage]: can we add host check here to only allow host=RHEL7 , guest = RHEL8 case? other case return 8 still an error?

Comment (wanxiangyuer) [reply zthin-parts/zthin/bin/unpackdiskimage]: It is easy to detect the operating system on the host. At the stage of unpackimage, the guest VM is only a partition. It is not easy to obtain the operating system version of a partition. @bjhuangr  and I tried many ways to check the operating system version on a partition. The only available method is to mount the partition to a directory, read the /etc/redhat-release file, get the operating system version, and then umount the directory.

Comment (jichenjc) [reply zthin-parts/zthin/bin/unpackdiskimage]: ok, then let's keep it as it is ..

**Pull request closed without merging**